qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -43,6 +43,15 @@
             type = virtio_serial_file_transfer_max_ports
             start_vm = no
             virtio_serial_ports = 30
+        - offline_migration:
+            only unix_socket
+            type = virtio_serial_file_transfer_offline_migrate
+            serials += " vs2"
+            serial_type_vs1 = virtserialport
```

Comment:
vs1 already be added with L6~L7.  no serial_type for the new serial vs2.
And could you explain the reason for adding vs2 ?

Reply:
sorry, it should be `serial_type_vs2` and already updated.

Reply:
@nanliu-r could you explain the reason of adding vs2 but do not use it?

Reply:
> @nanliu-r could you explain the reason of adding vs2 but do not use it?

To make the test more reasonable I will add a test of vs2, thanks.

Reply:
@nanliu-r As we discussed, test both vs1 and vs2 before migrate and after migrate, and update polarion case too.

Reply:
thanks and update.